CHANGELOG.md

```diff
@@ -37,6 +37,9 @@
         * Update `ThreeDSecureRequest.isDataOnlyRequested()` to `ThreeDSecureRequest.getDataOnlyRequested()`
         * Update `ThreeDSecureRequest.isExemptionRequested()` to `ThreeDSecureRequest.getDataOnlyRequested()`
         * Update `ThreeDSecureRequest.isCardAddChallengeRequested()` to `ThreeDSecureRequest.getCardAddChallengeRequested()`
+    *  ThreeDSecure
+      * Split `ThreeDSecureV2UiCustomization.setButtonCustomization()` to `setButtonCustomization()` and `setButtonType()`
```

Comment:
Any thoughts on this change?

I needed to break this setter apart to be able to use the `@Parcelize` annotation. The actual Cardinal function takes in both in the same setter. There shouldn't be any difference functionally.